src/Admin/MembreCentreAdmin.php

```diff
@@ -29,5 +31,14 @@ protected function configureFormFields(FormMapper $form): void
                 ],
             ])
             ->end();
+
+        $form->getFormBuilder()->addEventListener(FormEvents::POST_SET_DATA, function (FormEvent $event) {
+            $data = $event->getData();
+            if (!$data instanceof MembreCentre || $data->getId()) {
+                return;
+            }
+
+            $data->togglePermission(MembreCentre::MANAGE_BENEFICIARIES_PERMISSION);
```

Comment:
We should have a different method name `addPermission`, it will make the code clearer